src/Honeybee_addHBGlz.py

```diff
@@ -48,7 +48,7 @@
 
 ghenv.Component.Name = 'Honeybee_addHBGlz'
 ghenv.Component.NickName = 'addHBGlz'
-ghenv.Component.Message = 'VER 0.0.59\nJAN_26_2016'
+ghenv.Component.Message = 'VER 0.0.59\nJUNE_11_2016'
```

Comment:
This should be +ghenv.Component.Message = 'VER 0.0.59\nJUN_11_2016'. Remove the E at the end of the month.